### DISCUSSION TITLE
mypy compatibility

### DISCUSSION THREAD

**Discussion by gbrandt** [flet-dev/flet#743] [Q&A]
UPDATE:  ok, ignore this, type annotations are not complete so obviously it can't be advertised as such.

Looking through the source, it looks like everything is properly typed.  MyPy, however, complains because there is no `py.typed` file in the root directory.

Adding this to the root directory allows mypy (and I assume other type checkers) to run properly as per PEP 561.  My tests appear to be ok after adding `py.typed`.

Is there any reason this is not included with the distribution?

**Comment by FeodorFitsner** [comment]
Would you like to do a PR with that?
